modules/core/src/main/java/org/apache/fluo/core/impl/TxStringUtil.java

```diff
@@ -56,6 +56,7 @@ public static String gets(SnapshotBase snapshot, String row, Column column) {
     return transform(snapshot.get(Collections2.transform(rows, s -> Bytes.of(s)), columns));
   }
 
+  // TODO: Fix incompatable return type.
```

Comment:
Are you familiar w/ Lambda's in Java8? Could replace method with the following, which uses `Maps.transformValues()` and passes it a Lambda to transform the value from Bytes to String.

```java
public static Map<RowColumn, String> gets(SnapshotBase snapshot, Collection<RowColumn> rowColumns) {
    Map<RowColumn, Bytes> bytesMap = snapshot.get(rowColumns);
    return Maps.transformValues(bytesMap, b -> b.toString());
 }

```